src/includes/user-feedback/http-request/android.mdx

```diff
@@ -0,0 +1,26 @@
+```java
+import io.sentry.Sentry;
```

Comment:
The name `user-feedback/http-request` is a bit odd since these examples are showing the SDK API in that programming language. As opposed to the Web API added to the bottom of the common user feedback page:

> If you'd prefer an alternative to the widget or do not have a JavaScript frontend, you can use the [User Feedback API](/api/projects/submit-user-feedback/).

Reply:
Yes, I agree. I can change it to `user-feedback/sdk-api-example`